src/BuildScriptGenerator/PlatformInstallerBase.cs

```diff
@@ -182,6 +183,15 @@ public virtual void InstallPlatformSpecificSkeletonDependencies(StringBuilder st
                 .AppendLine($"mkdir -p /home/codespace/.ruby")
                 .AppendLine($"ln -sfn /opt/ruby/{version} /home/codespace/.ruby/current")
                 .AppendLine("fi")
+                .AppendLine($"if [ -f \"$oryxImageDetectorFile\" ] && [ -f \"$oryxOsDetectorFile\" ] && [ \"$platformName\" = \"python\" ] && grep -q \"githubactions\" \"$oryxImageDetectorFile\" && grep -q \"BULLSEYE\" \"$oryxOsDetectorFile\"; then")
```

Comment:
I'm thinking rather than having these commands in `PlatformInstallerBase.cs` file, would it make sense to put them under the corresponding build or runtime dockerfiles?

Reply:
@qianz2 the corresponding runtime images have these changes, so we're good there, but the corresponding bullseye images require them when the user is trying to build a Python 3.7 or 3.8 application, which we won't know until build time, and I didn't want to "bloat" the bullseye image with additional dependencies that may or may not be needed. I don't mind migrating them over if you feel the bullseye image Dockerfile would be a better location for it though.